simulation_ws/.rosinstall

```diff
@@ -1,5 +1,7 @@
 - git: {local-name: src/deps/aws-robomaker-bookstore-world, uri: "https://github.com/aws-robotics/aws-robomaker-bookstore-world", version: v0.0.1}
 - git: {local-name: src/deps/aws-robomaker-small-house-world, uri: "https://github.com/aws-robotics/aws-robomaker-small-house-world", version: ros1}
+- git: {local-name: src/deps/aws-robomaker-small-warehouse-world, uri: "https://github.com/aws-robotics/aws-robomaker-small-warehouse-world", version: master}
 - git: {local-name: src/deps/turtlebot3, uri: "https://github.com/ROBOTIS-GIT/turtlebot3", version: d3cdcc6647812ae9a83f05e626cdae322923ac84}
 - git: {local-name: src/deps/turtlebot3_simulations, uri: "https://github.com/ROBOTIS-GIT/turtlebot3_simulations", version: 1.2.0}
 - git: {local-name: src/deps/aws-robomaker-simulation-ros-pkgs, uri: 'https://github.com/aws-robotics/aws-robomaker-simulation-ros-pkgs.git', version: master}
+- git: {local-name: src/deps/map_generation_plugin, uri: 'https://github.com/ojasjoshi/gazebo_ros_2Dmap_plugin', version: melodic-devel}
```

Comment:
Is this a fork of an upstream repo? Are you able to contribute back upstream and use that instead of a personal fork?

Reply:
Yes it is a fork. I have created a upstream PR for this change but the original author has not updated yet.. it's in my backlog todo list

The author was responding till last week to PRs but I can push to update it faster if possible..

Reply:
Are you waiting on https://github.com/marinaKollmitz/gazebo_ros_2Dmap_plugin/pull/8? Is there anything else?

Reply:
Yep that's the only PR left to be merged upstream